api/src/main/java/io/strimzi/api/kafka/model/EntityTopicOperatorSpec.java

```diff
@@ -27,7 +27,7 @@
 @JsonInclude(JsonInclude.Include.NON_NULL)
 @JsonPropertyOrder({"watchedNamespace", "image",
         "reconciliationIntervalSeconds", "zookeeperSessionTimeoutSeconds",
-        "resources", "topicMetadataMaxAttempts", "logging"})
+        "resources", "topicMetadataMaxAttempts", "logging", "gcLoggingDisabled"})
```

Comment:
Surely this should be a property of `JvmOptions`?

Reply:
Why do you think this should be property of `jvmOptions` just here? Or do you mean this comment for all resources?